how-to/security_hardening.rst

```diff
@@ -0,0 +1,60 @@
+JAAS: Security Hardening
+========================
+JIMM, the service at the centre of JAAS can be hardened in a number of ways. This 
+document details how you can harden the security of your JAAS deployment. 
+
+CORS
+----
+To set CORS on JIMM, use the configuration option "cors-allowed-origins".
```

Comment:
Surround `cors-allowed-origins` with two backticks like this and you won't need to update the `.custom_wordlist.txt` file:

```
``cors-allowed-origins``
```

Reply:
ooh